docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -233,7 +233,7 @@ Standard TensorFlow\* operations:
 | Square| No |
 | Squeeze | The case when squeeze axis is not specified is not supported |
 | StopGradient | Not needed for shape inference |
-| StridedSlice | No |
+| StridedSlice | Supported only for constant begin, end, and strides inputs |
```

Comment:
Constant here is constant-foldable

Reply:
did you mean that here should've been written `supported only for constant-foldable begin...`?

Reply:
Changed `constant -> constatn-foldable` here
https://github.com/openvinotoolkit/openvino/pull/4367/files#diff-9aff84740d78af50e1351bb7064e05b918e7e7218f07eec668d48cf3b02ae5b1